### ISSUE TITLE
System.ArgumentNullException: 'Value cannot be null. '

### ISSUE THREAD

**Issue by BerndN68** [AutoMapper/AutoMapper.Extensions.OData#29]
The exception occure  if a query with select or expand operation, but no top or orderby operation is executed.

**Example:**
```
        [Fact]
        public async void BuildingSelectName_WithoutOrder_WithoutTop()
        {
            Test(await Get<CoreBuilding, TBuilding>("/corebuilding?$select=Name"));

            void Test(ICollection<CoreBuilding> collection)
            {
                Assert.Equal(4, collection.Count);
            }
        }
```

**Error message:**
Value cannot be null. (Parameter 'body')

Stack Trace:
[   at System.Dynamic.Utils.ContractUtils.RequiresNotNull(Object value, String paramName, Int32 index)
   at System.Dynamic.Utils.ExpressionUtils.RequiresCanRead(Expression expression, String paramName, Int32 idx)
   at System.Linq.Expressions.Expression.ValidateLambdaArgs(Type delegateType, Expression& body, ReadOnlyCollection`1 parameters, String paramName)
   at System.Linq.Expressions.Expression.Lambda[TDelegate](Expression body, String name, Boolean tailCall, IEnumerable`1 parameters)
   at System.Linq.Expressions.Expression.Lambda[TDelegate](Expression body, Boolean tailCall, IEnumerable`1 parameters)
   at System.Linq.Expressions.Expression.Lambda[TDelegate](Expression body, ParameterExpression[] parameters)
   at Trumpf.Fab.FabricationService.AutoMapperExentsionsOData.LinqExtensions.GetQueryableExpression[T](ODataQueryOptions`1 options) in C:\FAB_DevOps\TruTopsFab\Source\Services\Fabrication\Source\FabricationService\AutoMapperExentsionsOData\LinqExtensions.cs:line 64]([url](`url`))

**Possible solution:**
LinqExtension.cs Line 81
return expression instead of null

```
public static Expression GetOrderByMethod<T>(this ODataQueryOptions<T> options, Expression expression)
        {
            if (options.OrderBy == null && options.Top == null)
                return expression;

```

**Comment by BerndN68** [comment]
****

**Comment by BlaiseD** [comment]
Thanks

**Comment by BlaiseD** [comment]
Should be fixed in 1.0.4-preview01.  Please try the [MyGet build](https://myget.org/feed/automapperdev/package/nuget/AutoMapper.AspNetCore.OData.EFCore).
